<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HtmlGeneratorWidget</class>
 <widget class="QWidget" name="HtmlGeneratorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>HTML</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="iconPage">
      <layout class="QVBoxLayout" name="iconPageLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="presetIconView">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget {
	background-color: palette(window);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="movement">
          <enum>QListView::Movement::Static</enum>
         </property>
         <property name="resizeMode">
          <enum>QListView::ResizeMode::Adjust</enum>
         </property>
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="viewMode">
          <enum>QListView::ViewMode::IconMode</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="editorPage">
      <layout class="QVBoxLayout" name="editorPageLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="HtmlGenerator">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>437</width>
            <height>557</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="ServicePresetWidget" name="preset" native="true"/>
           </item>
           <item>
            <widget class="QLabel" name="textLabel">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="visible">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Text</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="textVerticalLayout">
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="line1HorizontalLayout">
               <item>
                <widget class="QLabel" name="line1Label">
                 <property name="text">
                  <string>Line 1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line1LineEdit"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="line2HorizontalLayout">
               <item>
                <widget class="QLabel" name="line2Label">
                 <property name="text">
                  <string>Line 2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line2LineEdit"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="line3HorizontalLayout">
               <item>
                <widget class="QLabel" name="line3Label">
                 <property name="text">
                  <string>Line 3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line3LineEdit"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="bodyToggleButton">
             <property name="toolTip">
              <string>Click to expand or collapse the Body section</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton { text-align: left; font-weight: bold; }</string>
             </property>
             <property name="text">
              <string>▶ Body</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="bodyVerticalLayout">
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="bodyHorizontalLayout">
               <item>
                <widget class="QPushButton" name="colorButton">
                 <property name="text">
                  <string>Background color...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="colorLabel">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>#00000000</string>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="toolTip">
                  <string>Click to set color to transparent</string>
                 </property>
                 <property name="text">
                  <string>Transparent</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="bodySpacerLabel">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>2</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string notr="true"/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="bodyTextEdit">
               <property name="placeholderText">
                <string>Do not include &lt;html&gt;, &lt;head&gt;, or &lt;body&gt; here</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="cssToggleButton">
             <property name="toolTip">
              <string>Click to expand or collapse the CSS section</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton { text-align: left; font-weight: bold; }</string>
             </property>
             <property name="text">
              <string>▶ CSS</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="cssVerticalLayout">
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPlainTextEdit" name="cssTextEdit">
               <property name="placeholderText">
                <string>Type or paste stylesheets here</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="javascriptToggleButton">
             <property name="toolTip">
              <string>Click to expand or collapse the JavaScript section</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton { text-align: left; font-weight: bold; }</string>
             </property>
             <property name="text">
              <string>▶ JavaScript</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="javascriptVerticalLayout">
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPlainTextEdit" name="javascriptTextEdit">
               <property name="placeholderText">
                <string>Type or paste JavaScript code here</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="imageButton">
           <property name="text">
            <string>Generate Image...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="videoButton">
           <property name="text">
            <string>Generate Video...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ServicePresetWidget</class>
   <extends>QWidget</extends>
   <header>widgets/servicepresetwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
